src/Agent.Worker/AgentPluginManager.cs

```diff
@@ -22,11 +22,11 @@ public interface IAgentPluginManager : IAgentService
         Task RunPluginTaskAsync(IExecutionContext context, string plugin, Dictionary<string, string> inputs, Dictionary<string, string> environment, Variables runtimeVariables, EventHandler<ProcessDataReceivedEventArgs> outputHandler);
     }
 
-    public sealed class AgentPluginManager : AgentService, IAgentPluginManager
+    public class AgentPluginManager : AgentService, IAgentPluginManager
```

Comment:
Same question about deriving.

Reply:
Not sure which question this is in reference to

Reply:
We want to derive here so that we can add/remove plugins without altering any of the rest of the execution